### DISCUSSION TITLE
When will Ubuntu 22.04 be supported

### DISCUSSION THREAD

**Discussion by rabol** [deployphp/deployer#3660] [General]
Hi

Ubuntu 22.04 has been out for a while so I was wondering when it will be supported ?

**Comment by antonmedv** [comment]
Yes, at it one of LTS. Also I’m planning to support newest versions as well.

**Reply by SimJoSt** [reply]
That sounds great. It would make provisioning with newer machines easier.

**Reply by alexandersix** [reply]
Is there any update on when 22.04 might be supported? The documentation still says that only servers running 20.04 are supported, so I assume 22 is still not supported.

**Reply by alokVishu** [reply]
Is there any update?

**Reply by antonmedv** [reply]
I will try to do an update after my vacation  🤞🏻

**Reply by stebogit** [reply]
Hello, any update? I'm particularly interested in the provisioning part.
The docs still [says](https://deployer.org/docs/7.x/getting-started#provision) only Ubuntu 20.04 is supported, however there are [old issues](https://github.com/deployphp/deployer/issues?q=is%3Aissue+%22Ubuntu+22%22) closed/fixed regarding Ubuntu 22.04.

**Reply by F-Yousri** [reply]
would be great if possible as aws only supports newer ubuntu versions now

**Reply by antonmedv** [reply]
Yes! You can try to use v7.5.0-beta right now!

**Comment by gbaudoin** [comment]
I'm using it on Ubuntu 22.04 with PHP 8.1. Apart from having to add php8.1-redis , it worked without errors.
